src/user_interfaces/gazebo_plugin.cpp

```diff
@@ -83,6 +129,8 @@ bool VoxbloxGroundTruthPlugin::serviceCallback(
                              << ". Currently, "
                              << "only triangular meshes are supported."
                              << "\nSkipping this mesh.";
+                  // TODO: Add support for other mesh types, e.g.
+                  //       - common::SubMesh::LINES
```

Comment:
While testing with an example world from the forest_gen repo, I've noticed that the LINES mesh type is not supported but mainly used in the models. This might be a good thing to add in the future, but idk how much effort it required.

Reply:
Ok, thanks! That's good to know, then we can keep it in mind.